app/Models/Mentee.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Mentee extends Model
+{
+    protected $fillable = [
+        'currentstatus',
+        'previousexp',
+        'mentortype',
+        'timeframe',
+        'suitabletime',
+        'extrainfo',
+        'status',
```

Comment:
What does the status on a Mentee represent? Is it different from the status we were talking about adding to the Mentorship model?

Reply:
Yes, this is the status of the profile itself, so different from the Mentorship status. -- There was a comment by John in the backend code to set up the profile as pending. I think this is about making sure nobody will post spoof profiles, as we once discussed at Framework.

Reply:
That makes sense, thanks! Will we need it for Mentors too?

Reply:
Yes, it is already on there :)